R/utils_data_handling.R

```diff
@@ -19,7 +19,7 @@
 #' @keywords data-handling
 #' @export
 #' @examples
-#' sample_to_quantile(as_forecast(example_sample_discrete))
+#' sample_to_quantile(as_forecast_sample(example_sample_discrete))
```

Comment:
I think we can see that this is the right choice here as trivial to make this an actual method of as_forecast_sample